### PR TITLE
[15.0][MIG] scheduler_error_mailer: Migration to version 15.0

### PR DESCRIPTION
**Pull request by ernesto-garcia-tecnativa**
@tecnativa TT41023

### CODE REVIEW

**Comment by ernesto-garcia-tecnativa** [comment]
@pedrobaeza  @victoralmau  please review

File: scheduler_error_mailer/data/ir_cron_email_tpl.xml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="0">
```
Comment (victoralmau) [inline scheduler_error_mailer/data/ir_cron_email_tpl.xml]:
```suggestion
<odoo noupdate="1">
```
Keep the same behaviour as before.

File: scheduler_error_mailer/models/ir_cron.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+
+class IrCron(models.Model):
+    _inherit = "ir.cron"
+
+    email_template_id = fields.Many2one(
+        comodel_name="mail.template",
```
Comment (victoralmau) [inline scheduler_error_mailer/models/ir_cron.py:18]:
```suggestion
        comodel_name="mail.template",
        domain=[("model_id.model", "=", "ir.cron")],
```
We can use the opportunity to add a domain.

**Review by victoralmau — CHANGES_REQUESTED** [review]
Good work, just a few minor changes.

**Comment by ernesto-garcia-tecnativa** [comment]
> Good work, just a few minor changes.

Thank you, I push a new commit with your suggestions

**Comment by victoralmau** [comment]
> > Good work, just a few minor changes.
> 
> Thank you, I push a new commit with your suggestions

Not necessary, all changes must be in the migration commit.

**Comment by ernesto-garcia-tecnativa** [comment]
> > > Good work, just a few minor changes.
> > 
> > 
> > Thank you, I push a new commit with your suggestions
> 
> Not necessary, all changes must be in the migration commit.

Ready, I wait the tests

**Review by victoralmau — APPROVED** [review]
Code review OK.

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**